apiserver/Monitoring.md

```diff
@@ -0,0 +1,71 @@
+# Monitoring of the API server and created Ray clusters
+
+In order to ensure a proper functioning of the API server and created Ray clusters, it is typically necessary to
+monitor the API server itself and created Ray clusters. This document describes how to instrument both API server
+and created clusters with Prometheus and Grafana
+
+## Monitoring of the API server
+
+Current implementation of the API server provides a flag `collectMetricsFlag` that defines whether to collect and
+expose Prometheus metrics, which is by default set to `true`. To disable metrics collection, this flag needs to
+be set to `false` and the API server image needs to be rebuild.
+
+If this flag is enabled, an `http` port at `/metrics` endpoint provides API server metrics in Prometheus format.
+
+### Testing monitoring of the API server
+
+On the kind server Install Kubernetes Prometheus Stack via Helm chart using the following
+[command](../install/prometheus/install.sh). The script additionally creates `ray-head-monitor` and
+`ray-workers-monitor` in the `prometheus-system` namespace, that we do not need. We can delete them using:
+
+```shell
+kubectl delete servicemonitor ray-head-monitor -n prometheus-system
+kubectl delete podmonitor ray-workers-monitor -n prometheus-system
+```
+
+Install API server from the project using the following command:
+
+```shell
+helm install apiserver /Users/boris/Projects/kuberay/helm-chart/kuberay-apiserver
```

Comment:
why not use the instructions from the 'README.md' on how to deploy the apiserver? (and now the whole world know where on your laptop you keep this source code :))

Reply:
I wanted to emphasize that you need to install it on the cluster, but yes, you are right. Pls, suggest rephrasing this

Reply:
How about something allong these lines:

```text
To install API server please follow the instructions from the [README.md](https://github.com/ray-project/kuberay/blob/master/apiserver/README.md#helm)
```

Reply:
Done